packages/communication-react/review/communication-react.api.md

```diff
@@ -2292,7 +2292,7 @@ export type VideoGalleryParticipant = {
 // @public
 export interface VideoGalleryProps {
     dominantSpeakers?: string[];
-    layout?: 'default' | 'floatingLocalVideo';
+    layout?: 'default' | 'floatingLocalVideo' | /* @conditional-compile-remove-from(stable) meeting/calling-composite <Local-Camera-Switcher> */ 'controlsLocalVideo';
```

Comment:
`controlsLocalVideo` is difficult to parse meaning from - what controls and how does it differ from the other options? ...though I'm drawing a blank at a good solid name, will have a think...

Reply:
I was thinking on this name too, I could go more specific to something like `LocalCameraButtonLocalVideo` since thats what its for but went with `controlsLocalVideo` since in the future we might add different controls? does that maybe help with thinking of something better?